### PR TITLE
Correct supermarket and github inspec.yaml examples

### PR DESCRIPTION
**Pull request by smith**
Signed-off-by: Nathan L Smith <smith@chef.io>

### CODE REVIEW

File: docs/profiles.md [hunk 1]
```diff
@@ -177,7 +175,7 @@ For example:
 
     depends:
     - name: gh-profile
-      git: username/project
+      github: username/project
       branch:  desired_branch
```
Comment (chris-rock) [inline docs/profiles.md]: branch, tag and commit are not valid. If they need to be pinned you should switch to the `git` fetcher. This is the explanation:
```
    # Transforms a browser github url to github tar url
    # We distinguish between three different Github URL types:
    #  - Master URL
    #  - Branch URL
    #  - Commit URL
    #
    # master url:
    # https://github.com/nathenharvey/tmp_compliance_profile/ is transformed to
    # https://github.com/nathenharvey/tmp_compliance_profile/archive/master.tar.gz
    #
    # github branch:
    # https://github.com/hardening-io/tests-os-hardening/tree/2.0 is transformed to
    # https://github.com/hardening-io/tests-os-hardening/archive/2.0.tar.gz
    #
    # github commit:
    # https://github.com/hardening-io/tests-os-hardening/tree/48bd4388ddffde68badd83aefa654e7af3231876
    # is transformed to
    # https://github.com/hardening-io/tests-os-hardening/archive/48bd4388ddffde68badd83aefa654e7af3231876.tar.gz
```

Comment (chris-rock) [reply docs/profiles.md]: Therefore you can use:
- dev-sec/linux-baseline
- dev-sec/linux-baseline/tree/2.0
- dev-sec/linux-baseline/tree/48bd4388ddffde68badd83aefa654e7af3231876
as valid github urls

Comment (smith) [reply docs/profiles.md]: Ok will change this. Looks like this was copied from the `git` section above and I just left it.

Comment (smith) [reply docs/profiles.md]: Fixed in 236c516.

**Comment by chris-rock** [comment]
Since we are at it, should we give examples for:
- dev-sec/linux-baseline
- dev-sec/linux-baseline/tree/2.0
- dev-sec/linux-baseline/tree/48bd4388ddffde68badd83aefa654e7af3231876
as valid github urls
as well?

**Comment by smith** [comment]
> Since we are at it, should we give examples for:
> ...
> as well?

Sure. Added in b169bec.

**Review by adamleff — APPROVED** [review]
Thanks a lot for the updates, @smith.

**Pull request merged**